sang-kun/github-profile/github-profile/src/components/DarkMode.js

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect, useState } from "react";
+
+import "../App.css";
+
+const DarkModeToggle = () => {
+  const [checked, setChecked] = useState(
+    localStorage.getItem("theme") === "dark" ? true : false
+  );
+  useEffect(() => {
```

Comment:
Good! For your next project, let think about how to make custom hook to make code cleaner